### PR TITLE
Merge `9.0.x` into `develop`

### PR DESCRIPTION
**Pull request by boherm**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Merge `9.0.x` into `develop`
| Type?             | improvement
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI & UI Tests 🟢
| UI Tests          | https://github.com/boherm/ga.tests.ui.pr/actions/runs/12178030181
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Review by nicosomb — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
